combat/warriors.cpp

```diff
@@ -767,7 +919,7 @@ int cmdBash(const std::shared_ptr<Player>& player, cmd* cmnd) {
 
 int cmdKick(const std::shared_ptr<Player>& player, cmd* cmnd) {
     std::shared_ptr<Creature> creature;
-    long    i, t;
+    long    i,j,t;
```

Comment:
+1 descriptive naming

Reply:
Changed i to lt_kick, and j to lt_gore. t is used as time(0) throughout the code.